TestSwitchApi/TestSwitchApi/DataMigrations/001_InitialSchemaSetup.sql

```diff
@@ -58,6 +58,18 @@ ALTER TABLE ONLY public."Candidates" ALTER COLUMN "Id" SET DEFAULT nextval('publ
 INSERT INTO public."Candidates" ("FirstName", "LastName", "Email", "Id") VALUES ('Peter', 'Parker', 'amazing@web.net', 1);
 INSERT INTO public."Candidates" ("FirstName", "LastName", "Email", "Id") VALUES ('Faye', 'Valentine', 'myfunny@valentine.com', 2);
 INSERT INTO public."Candidates" ("FirstName", "LastName", "Email", "Id") VALUES ('Bruce', 'Wayne', 'darkKnight@gotham.com', 3);
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Jeffie', 'Norton', 'JNorton@web.net');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Lettie', 'Jalbert', 'LJalbert@valentine.com');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Gladis', 'Mcnatt', 'GMcnatt@gotham.com');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Merlin', 'Montesino', 'MMontesino@web.net');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Manuel', 'Berta', 'MBerta@valentine.com');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Janene', 'Stogner', 'JStogner@gotham.com');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Christian', 'Barba', 'CBarba@web.net');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Ward', 'Hawes', 'WHawes@valentine.com');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Sulema', 'Tussey', 'STussey@gotham.com');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Marvella', 'Smythe', 'MSmythe@web.net');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Phylis', 'Spengler', 'PSpengler@valentine.com');
+INSERT INTO public."Candidates" ("FirstName", "LastName", "Email") VALUES ('Alia', 'Tall', 'ATall@gotham.com');
```

Comment:
Don't ever edit a migration once its been committed.

As soon as its been committed to master, its been run on staging / production / other people's machines.
If we change it, then everyone else has to manually undo it and reapply, or work out what's changed or just miss out on the new stuff - very quickly becomes a giant mess! :) 

Much better to generate a brand new migration - this way its easy for everyone to stay up to date.

(In general though I'd be careful about having data in our migrations - we don't really want all this fake data in the real app)

Reply:
Have removed all the insert commands with fake data.